lib/fedex/request/shipment.rb

```diff
@@ -46,7 +46,7 @@ def add_requested_shipment(xml)
           add_shipper(xml)
           add_recipient(xml)
           add_shipping_charges_payment(xml)
-          add_special_services(xml) if @shipping_options[:return_reason] || @shipping_options[:cod]
+          add_special_services(xml) if @shipping_options[:return_reason] || @shipping_options[:cod] || @shipping_options[:saturday_delivery]
```

Comment:
hmmm special services options is growing. I think we could pass it as an additional hash(which would be optional). I have plans to refactor a lot of the code in the near future so I'm okay merging as is knowing we need to re-think how special services should behave.